### ISSUE TITLE
NTR: Lipoprotein(a) high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8306]
New term request Lipoprotein(a) high in blood (elthat_130207182916): LPA; Lp(a)

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000699
- label: Elevated circulating lipoprotein(a) concentration
- parents: [Increased circulating apolipoprotein circulation (HP:0033460)](https://hpo.jax.org/app/browse/term/HP:0033460)
- pmid: PMID:10484779
- def: The concentration of lipoprotein(a) in the blood circulation is above the upper limit of normal.
- comment: Lipoprotein(a) [Lp(a)] is a macromolecular complex in human plasma which represents a quantitative genetic trait with a heritability of 0.7 or higher. High concentrations of Lp(a) are associated with atherothrombotic disease i.e. myocardial infarction, stroke, peripheral vascular disease and childhood thromboembolism. The covalent Lp(a) complex is assembled in plasma from low-density lipoprotein (LDL) and apolipoprotein(a), a plasminogen-related glycoprotein.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8306
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal
Please note because of an error, the termid was wrong. It should be
HP:6000521
